ui/components/app/nfts-tab/nfts-tab.js

```diff
@@ -88,137 +104,112 @@ export default function NftsTab() {
   }
 
   return (
-    <Box className="nfts-tab">
-      {hasAnyNfts > 0 || previouslyOwnedCollection.nfts.length > 0 ? (
-        <NftsItems
-          collections={collections}
-          previouslyOwnedCollection={previouslyOwnedCollection}
-        />
-      ) : (
-        <>
-          {isMainnet && !useNftDetection ? (
-            <Box paddingTop={4} paddingInlineStart={4} paddingInlineEnd={4}>
-              <NFTsDetectionNoticeNFTsTab />
-            </Box>
-          ) : null}
-          {
-            ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
-            showNftBanner ? (
-              <Box
-                paddingInlineStart={4}
-                paddingInlineEnd={4}
-                display={Display.Flex}
-                paddingTop={4}
-              >
-                <AssetListConversionButton
-                  variant={ASSET_LIST_CONVERSION_BUTTON_VARIANT_TYPES.NFT}
-                  onClick={() => {
-                    global.platform.openTab({ url: ZENDESK_URLS.NFT_TOKENS });
-                    trackEvent({
-                      event: MetaMetricsEventName.EmptyNftsBannerClicked,
-                      properties: {
-                        chain_id: chainId,
-                        locale: currentLocale,
-                        network: nickname,
-                        referrer: ORIGIN_METAMASK,
-                      },
-                    });
-                  }}
-                />
+    <>
+      {showRampsCard && <RampsCard variant={RAMPS_CARD_VARIANT_TYPES.NFT} />}
+      <Box className="nfts-tab">
+        {hasAnyNfts > 0 || previouslyOwnedCollection.nfts.length > 0 ? (
+          <NftsItems
+            collections={collections}
+            previouslyOwnedCollection={previouslyOwnedCollection}
+          />
+        ) : (
+          <>
+            {isMainnet && !useNftDetection ? (
+              <Box paddingTop={4} paddingInlineStart={4} paddingInlineEnd={4}>
+                <NFTsDetectionNoticeNFTsTab />
               </Box>
-            ) : null
-            ///: END:ONLY_INCLUDE_IF
-          }
-          <Box
-            padding={12}
```

Comment:
FYI, git is confused at these changes because of the indent and the diff is incorrect. The JSX in this file changed from:

```
 <Box className="nfts-tab">
   <All other content />
 </Box>
```
to this:

```
<>
  <RampsCard/>
  <All other content />
</>
```

The only exception is the removal of the <AssetListConversionButton/>

Reply:
Hiding whitespace changes helps a lot here (or adding this query param to the url `w=1`)